src/android/TP6m.jsx

```diff
@@ -36,7 +37,9 @@ class TP6M extends React.Component {
   }
 
   async componentDidMount() {
-    const { test, platform } = this.props;
+    const { test, platform, past, ending } = this.props;
+    // BE SURE THE timeDomain IS SET BEFORE DOING ANY await
+    const timeDomain = new TimeDomain({ past, ending });
```

Comment:
This is a common theme: Converting the `past`, `ending` pair into a TimeDomain object